pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -49,14 +65,19 @@ func main() {
 		"package embeddedyamls\n\nconst (\n")
 	panicOnErr(err)
 
+	re, err := regexp.Compile("`([^`]*)`")
+	panicOnErr(err)
+
 	for _, f := range files {
 
 		_, err = out.WriteString("\t" + constName(f) + " = `")
 		panicOnErr(err)
 
 		fmt.Println(f)
-		f, _ := os.Open(path.Join(yamlsDirectory, f))
-		_, err = io.Copy(out, f)
+		contents, err := ioutil.ReadFile(path.Join(yamlsDirectory, f))
+		panicOnErr(err)
+
```

Comment:
Can we add a comment explaining why we do this? I guess it’s related to comments , strings or annotations or something?